ApplicationLibCode/Commands/RicCreateEnsembleWellLogFeature.cpp

```diff
@@ -287,9 +288,19 @@ RimEclipseCase* RicCreateEnsembleWellLogFeature::loadEclipseCase( const QString&
     }
     bool createView = false;
     bool createPlot = false;
+
+    std::shared_ptr<RifReaderSettings> readerSettings = std::make_shared<RifReaderSettings>();
+    // Disable as much as possible
+    readerSettings->importNNCs                          = false;
```

Comment:
Add a method used to populate these settings

Reply:
Fixed.